lua/telescope/builtin/__lsp.lua

```diff
@@ -359,7 +359,11 @@ lsp.document_symbols = function(opts)
       return
     end
 
-    local locations = vim.lsp.util.symbols_to_items(result or {}, opts.bufnr) or {}
+    if vim.fn.has "nvim-0.11" == 1 then
+      local locations = vim.lsp.util.symbols_to_items(result or {}, opts.bufnr, "utf-8") or {}
+    else
+      local locations = vim.lsp.util.symbols_to_items(result or {}, opts.bufnr) or {}
+    end
```

Comment:
Also, the `locations` variable is local to the `if` and `else` block, they're not visible outside them. You'll need to define the variable before the `if` statement:
```lua
local locations
if ... then
	locations = ...
else
	locations = ...
end
```